CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v0.16.2 (2017-6-27)
+
+### Bug Fixes
+
+* Typing text now raises the `onChange` event in latest React versions. ([#1558](https://github.com/DevExpress/testcafe/issues/1558))
+* Screenshots can now be taken when TestCafe runs from the Docker image. ([#1540](https://github.com/DevExpress/testcafe/issues/1540))
+* The native `value` property setters of HTMLInputElement and HTMLTextAreaElement prototypes are now saved. ([testcafe-hammerhead/#1185](https://github.com/DevExpress/testcafe-hammerhead/issues/1185))
```

Comment:
Please highlight `HTMLInputElement`, `HTMLTextAreaElement`,  `HTMLCollection`, because they are the objects of DOM API